rules/common.js

```diff
@@ -26,18 +25,19 @@ module.exports = {
     'no-unused-vars': 1,
     semi: [1, 'always'],
     'import/no-unresolved': 1,
-    'import/prefer-default-export': 1,
+    'import/prefer-default-export': 0,
     'prefer-destructuring': 1,
     'lines-between-class-members': 1,
-    'consistent-return': 1,
-    'max-lines': ['warn', { max: 250, skipBlankLines: true, skipComments: true }],
-    'max-classes-per-file': ['warn', 3],
-    'max-lines-per-function': ['warn', { max: 75 }],
+    'max-lines': ['warn', { max: 500, skipBlankLines: true, skipComments: true }],
```

Comment:
At 500 lines this is point, might as well remove it. The idea is to extract common functionality, push single responsibility, encapsulate logic, etc. This is useless.